advanced/Scripts/version.sh

```diff
@@ -97,22 +112,48 @@ getRemoteVersion(){
     return 0
 }
 
+getLocalBranch(){
+    # Get the checked out branch of the local directory
+    local directory="${1}"
+    local branch
+
+     # Local FTL btranch is stored in /etc/pihole/ftlbranch
+    if [[ "$1" == "FTL" ]]; then
+        branch=$(cat /etc/pihole/ftlbranch)
```

Comment:
This file is not available if the user has never done a local checkout of a custom FTL branch, the output would be
```plain
cat: /etc/pihole/ftlbranch: No such file or directory
  Current FTL version is  v4.3.1-800-g853261bb.
```

Use
```suggestion
        branch="$(pihole-FTL branch)"
```
to get the branch of the installed FTL instance reliably (`pihole-FTL -b` is the short form but it doesn't hurt to be explicit).

Reply:
I pushed this change already, feel free to resolve this conversation if you agree.